config/nightwatch.js

```diff
@@ -38,7 +37,6 @@ module.exports = {
         acceptSslCerts: true,
         webStorageEnabled: true,
         chromeOptions: {
-          binary: electron,
```

Comment:
Removing this makes the e2e tests start up Chrome instead of Electron.

Reply:
Thanks @jbalboni, as mentioned in the PR description, there appears to be a bug when using electron related to resizeWindow, should chromedriver be installed/configured instead, or should these tests be put on the back burner for now, or is another solution preferred?

Reply:
Without doing any kind of digging into this, I think this is _probably_ fine, but I'd like a larger discussion before changing the browser we're running all e2e tests on.

Reply:
Sounds good @cvalarida, I'll remove the test related changes from the PR.